tasks/condition/condition.js

```diff
@@ -23,6 +30,28 @@ let task2;
 const alpha = 3;
 const beta = 7;
 const operator = "add";
+switch (operator) {
+  case operator === "add":
```

Comment:
you should not use === operator here:
````
case "add":
  task2 == alpha + beta;
  break;
````

Please do that for other cases as well